js/salesrecord.js

```diff
@@ -0,0 +1,5 @@
+const art = document.querySelector('article');
+const nav = document.querySelector('header');
+art.onscroll = () => {
+    console.log(window.art.scrollY);
```

Comment:
Do you need the console statement here?

Reply:
no just checking for a return value from that method will delete now